helm-charts/orchestrator/templates/sonataflows.yaml

```diff
@@ -32,6 +32,15 @@ spec:
         limits:
           memory: {{ .Values.orchestrator.sonataflowPlatform.resources.limits.memory }}
           cpu: {{ .Values.orchestrator.sonataflowPlatform.resources.limits.cpu }}
+  {{- if (and (.Values.orchestrator.sonataflowPlatform.broker.name) (.Values.orchestrator.sonataflowPlatform.broker.namespace)) }}
+  eventing:
+    broker:
+      ref:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        name: {{ .Values.orchestrator.sonataflowPlatform.broker.name }}
```

Comment:
shouldn't it be added to https://github.com/rhdhorchestrator/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/values.yaml#L95 ?
also, the above section 
```
              orchestrator:
                description: Orchestrator contains the fields for the orchestrator
                  configuration
                properties:
                  namespace:
                    default: sonataflow-infra
                    description: Namespace where sonataflow's workflows run. The value
                      is captured when running the setup.sh script and stored as a
                      label in the target namespace. User can override the value by
                      populating this field. Defaults to `sonataflow-infra`. This
                      field takes precendence over the value in the label.
                    type: string
                  sonataflowPlatform:
                    description: SonataflowPlatform contains the pod resource configuration
                      to be used for the data index and job services
                    properties:
                     ...
                      eventing:
                        description: SonataflowPlatform's eventing related properties 
                        properties:
                          broker:
                            description: Broker contains the name and namespace of the broker to use if using Knative eventing for components communication 
                            properties:
                              name:
                                description: Name describes the name of the broker to use
                                type: string
                              namespace:
                                description: Namespace describes the namespace on which the broker to use is deployed
                                type: string
                            type: object
```
but in the path above `.Values.orchestrator.sonataflowPlatform.broker.name` the `eventing` isn't mentioned.

Reply:
good catch, the structure evolved and I missed to update those!